package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.6.1",
   "description": "A human-friendly standard for Flux action objects",
   "main": "lib/index.js",
+  "typings": "src/index.d.ts",
   "files": [
     "lib"
```

Comment:
Doesn't the `index.d.ts` need to be in the published `files`? I would rather you move this to the root so that this array can just be: 

```json
[
  "lib",
  "index.d.ts"
]
```